cmd/csaf_checker/main.go

```diff
@@ -29,6 +31,7 @@ type options struct {
 	Insecure   bool    `long:"insecure" description:"Do not check TLS certificates from provider"`
 	ClientCert *string `long:"client-cert" description:"TLS client certificate file (PEM encoded data)" value-name:"CERT-FILE"`
 	ClientKey  *string `long:"client-key" description:"TLS client private key file (PEM encoded data)" value-name:"KEY-FILE"`
+	Version    bool    `long:"version" short:"v" description:"Display version of the binary"`
```

Comment:
According to https://github.com/csaf-poc/csaf_distribution/issues/66#issuecomment-1055111262 we do not want to have a short version for this option.